tests/relay/schema.gql

```diff
@@ -0,0 +1,189 @@
+input CreateFruitAsyncInput {
+  name: String!
+  color: String!
+}
+
+input CreateFruitInput {
+  name: String!
+  color: String!
+}
+
+type Fruit implements Node {
+  """The Globally Unique ID of this object"""
+  id: GlobalID!
+  name: String!
+  color: String!
+}
+
+type FruitAsync implements Node {
+  """The Globally Unique ID of this object"""
+  id: GlobalID!
+  name: String!
+  color: String!
+}
+
+"""A connection to a list of items."""
+type FruitAsyncConnection {
+  """Pagination data for this connection"""
+  pageInfo: PageInfo!
+
+  """Contains the nodes in this connection"""
+  edges: [FruitAsyncEdge!]!
+
+  """Total quantity of existing nodes"""
+  totalCount: Int
```

Comment:
can we remove this field by default? I don't think it's in the spec 😊
Maybe we can have specialised connections for this?